src/main/java/fruite/store/service/strategy/OperationType.java

```diff
@@ -0,0 +1,5 @@
+package fruite.store.service.strategy;
+
+public interface OperationType {
+    void doOpearation(String fruit, Integer quantity);
+}

```

Comment:
let's rename this class to OperationHandler, do the same for its implementations (BalanceOperationHandler, etc.)